pkg/regexp/regexp.go

```diff
@@ -10,26 +10,26 @@ import (
 // used as global variables. Using this structure helps speed the startup time
 // of apps that want to use global regex variables. This library initializes them on
 // first use as opposed to the start of the executable.
-type Regexp struct {
-	once   *sync.Once
+type Regexp = *regexpStruct
+
+type regexpStruct struct {
+	_      noCopy
```

Comment:
I have verified, at least going by `unsafe.Sizeof`, that this has no run-time memory cost.